homeassistant/components/switchbot/coordinator.py

```diff
@@ -55,6 +59,31 @@ def __init__(
         self._ready_event = asyncio.Event()
         self._was_unavailable = True
 
+    @callback
+    def _needs_poll(
+        self,
+        service_info: bluetooth.BluetoothServiceInfoBleak,
+        seconds_since_last_poll: float | None,
+    ) -> bool:
+        # Only poll if hass is running, we need to poll,
+        # and we actually have a way to connect to the device
+        return (
+            self.hass.state == CoreState.running
+            and self.device.poll_needed(seconds_since_last_poll)
+            and bool(
+                bluetooth.async_ble_device_from_address(
+                    self.hass, service_info.device.address, connectable=True
+                )
+            )
+        )
+
+    async def _async_update(
+        self, service_info: bluetooth.BluetoothServiceInfoBleak
+    ) -> dict[str, Any]:
+        """Poll the device."""
+        _LOGGER.warning("%s: Doing update", self.device_name)
```

Comment:
Is this the right log level for this op?

Reply:
Good catch. It's debug left in I thought I removed but it came back with the merge squash and rebase.

Reply:
Deleted 👍